certbot/certbot/_internal/main.py

```diff
@@ -722,8 +722,18 @@ def update_account(config, unused_plugins):
     # so that we can also continue to use the account object with acmev1.
     acc.regr = acc.regr.update(uri=prev_regr_uri)
     account_storage.update_regr(acc, cb_client.acme)
-    eff.prepare_subscription(config, acc)
-    add_msg("Your e-mail address was updated to {0}.".format(config.email))
+
+    if config.email is None:
+        add_msg(
+            ("Your e-mail address was removed. "
+            "Make sure you backup your account key stored in {0}\n".format(
```

Comment:
I think we should remove this sentence as I think ACME accounts are pretty disposable with the current spec.

I think the main concern with removing all of your contact information is, well, the CA cannot contact you so you won't get notifications about things like upcoming certificate expiration, revocation events, API changes, etc.

You could try to explain something like this in a CA agnostic way if you want, but I think just removing the sentence is fine since the damage is already done and this is explained in the `--register-unsafely-without-email` docs which the user had to provide.

Reply:
That makes sense - I got the text from the warning that `get_email()` [diplays](https://github.com/certbot/certbot/blob/master/certbot/certbot/display/ops.py#L34-L38) when you use `--register-unsafely-without-email` and I wanted to be consistent. Even the line about the email address being removed isn't a reliable description of what has happened - maybe I will change it to "Your account is no longer associated with an email address" as that will always be true.

Reply:
Ah thanks for the link! I didn't realize that text was there. It's outdated though so I don't think we should duplicate it here.

Your change to the first sentence SGTM!

Reply:
I realized that this will actually remove *all* contact information, including any phone numbers the user may have added. I used a version of the language that informed them of that.